hive/agents/qnets/utils.py

```diff
@@ -20,9 +20,18 @@ def calculate_output_dim(net, input_shape):
     """
     if isinstance(input_shape, int):
         input_shape = (input_shape,)
-    placeholder = torch.zeros((0,) + tuple(input_shape))
+    placeholder = torch.zeros((1,) + tuple(input_shape))
```

Comment:
why was this changed?

Reply:
The first dimension is the batch size. Technically it doesn't affect the output dim since extract_shape truncates the output. I remember we changed this dimension to 1 from 0 because the output hidden state had wrong dimensions. @hnekoeiq was that the reason?

Reply:
If I remember correctly, I had to do this because of RNN module.